controllers/api-controller.js

```diff
@@ -0,0 +1,11 @@
+const { allEndpoints } = require("../models/api-model");
+
+exports.getEndpoints = (req, res, next) => {
```

Comment:
Require in the endpoints data into the controller file then just send it back in the controller function